instructure/canvas-ios
# Fix Teacher ModuleItemSequence crash

refs: MBL-16730
affects: Teacher
release note: Fixed an occasional crash when browsing Module items 
test plan: 
1. Launch the Teacher app on iPad (or split view capable iPhone) using iOS 16
2. Open a course, Modules, select an Assignment
3. Edit button should be at the top right corner
4. Open a second app in split view 
5. Rotate the device back and forth 
6. Go back to full screen 
7. App shouldn't crash, Edit button should be visible 

## Checklist
